blockchain/genesis.go

```diff
@@ -329,7 +329,7 @@ func (g *Genesis) ToBlock(baseStateRoot common.Hash, db database.DBManager) *typ
 	}
 	// TODO-klaytn should set genesis block's base fee
 	if g.Config != nil && g.Config.IsKIP71ForkEnabled(common.Big0) {
-		head.BaseFee = new(big.Int).SetUint64(g.Config.Governance.KIP71.LowerBoundBaseFee)
+		head.BaseFee = new(big.Int).SetUint64(params.DefaultLowerBoundBaseFee)
```

Comment:
So if activate KIP-71 hard fork, then baseFee of genesis block will be always `params.DefaultLowerBoundBaseFee` right?
But when calculate next base fee, governance lower/upper bound will be used, so even `params.DefaultLowerBoundBaseFee` is not in lower/upper bound range, baseFee will be valid (inside of that range). 
Did i understand correctly?

Reply:
Sounds good!